mindsdb/interfaces/database/integrations.py

```diff
@@ -411,41 +399,6 @@ def _make_handler_args(self, name: str, handler_type: str, connection_data: dict
 
         return handler_ars
 
-    def create_tmp_handler(self, handler_type: str, connection_data: dict) -> object:
```

Comment:
this function is used, for example [here](https://github.com/mindsdb/mindsdb/blob/11b34a7eac44cd8c50754a6895ee1e1db74bf3d8/mindsdb/api/http/namespaces/config.py#L151)

Reply:
This is only for the tests. Do we need this since it brings a lot of complexity with it, which is not required now?

Reply:
@ZoranPandovski  yes, 'test connection' does not work anymore 
![image](https://github.com/mindsdb/mindsdb/assets/4068133/197990cb-a605-4084-80e0-591ffc945080)

Reply:
> which is not required now?

@ZoranPandovski  you mean not required in new cloud? but what about opensource version - should it have 'test connection' ability?

Reply:
@StpMax , @ea-rus I've changed the `create_tmp_handler` class to not use the `DBClient`, which was quite complex, and used grpc dependencies.